src/scanners/css.js

```diff
@@ -26,7 +26,15 @@ export default class CSSScanner extends BaseScanner {
                 }
 
                 this.validatorMessages = this.validatorMessages.concat(
-                  rules[cssRule](rule));
+                  rules[cssRule](
+                    rule.declarations,
```

Comment:
The `cssRule` object in this case is a notion of a "CSS Scannable Document", more/less right?

In that case I agree; we pass an HTML/RDF document to those rules.